File: App/app/src/main/java/cl/ucn/dics/pdis/appparkingucn/MainActivity.java
package cl.ucn.dics.pdis.appparkingucn;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.view.View;
import android.widget.*;
import android.os.Bundle;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button busquedaMain = findViewById(R.id.Buscar);
        Button registrarMain = findViewById(R.id.Registrar);

        setUiTransition(busquedaMain, registrarMain);
    }

    private void setUiTransition(Button busquedaMain, Button registrarMain){

        busquedaMain.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, Buscar.class));
            }
        });

        registrarMain.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, RegistroVehiculo.class));
            }
        });
    }
}

File: App/app/src/main/java/cl/ucn/disc/pdis/app/MainActivity.kt
package cl.ucn.disc.pdis.app

import android.content.Intent
import cl.ucn.disc.pdis.app.Buscar
import cl.ucn.disc.pdis.app.RegistroVehiculo
import com.zeroc.Ice.*
import android.os.Bundle
import android.widget.*
import androidx.appcompat.app.AppCompatActivity
import androidx.compose.Composable
import androidx.ui.core.setContent
import androidx.ui.foundation.Text
import androidx.ui.layout.Column
import androidx.ui.tooling.preview.Preview
import cl.ucn.disc.pdis.app.ui.AppTheme

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val busquedaMain = findViewById<Button>(R.id.Buscar)
        val registrarMain = findViewById<Button>(R.id.Registrar)
        setUiTransition(busquedaMain, registrarMain)
    }

    private fun setUiTransition(busquedaMain: Button, registrarMain: Button){

        busquedaMain.setOnClickListener {
            startActivity(Intent(this, Buscar::class.java))
        }

        registrarMain.setOnClickListener {
            startActivity(Intent(this, RegistroVehiculo::class.java))
        }
    }
}

